### ISSUE TITLE
DB Warning "accelerate does not support gradient_accumulation_steps when training multiple models"

### ISSUE THREAD

**Issue by IdiotSandwichTheThird** [kohya-ss/sd-scripts#76]
When running the db training script with gradient accumulation it warns `accelerate does not support gradient_accumulation_steps when training multiple models (U-Net and Text Encoder)`

But this same warning does not appear when using the finetune script.
Is this an actual issue for both scripts or just the db one?

Also I'm not even training the text encoder, so I'm wondering if I should be concerned at all?
